doc/advanced-testing-mini-course/README.md

```diff
@@ -16,9 +19,9 @@
 
 ```haskell
 data Network = Network
-  { deploy  :: Addr -> IO Socket -- bind and listen
-  , send    :: [(Addr, Msg)] -> IO ()
-  , select  :: [Socket] -> IO (Addr, Msg, Time) -- accept and recv
+  { deploy  :: Addr -> IO () -- bind and listen
+  , connect :: Addr -> IO ()
+  , select  :: [(Addr, Msg)] -> IO (Addr, Msg, Time) -- send, accept and recv
```

Comment:
Hmm, maybe we only change these if you correctly update `eventLoop` below?

Reply:
Fixed, thanks!